src/tasks/start/backend/update-app.ts

```diff
@@ -17,16 +17,16 @@ export async function updateApp(
   // Deploy a new app implementation.
   const implementation = await deployImplementation(bre.artifacts)
 
-  // Update the proxy with the new implementation.
-  await _updateProxy(implementation.address, appId, dao, bre.web3)
-
   // Update the repo with the new implementation.
   const { version, uri } = await _updateRepo(
     repo,
     implementation.address,
     appServePort
   )
 
+  // Update the proxy with the new implementation.
```

Comment:
This is subtle, but important! Some apps were failing because of updating the proxy first and then the repo. It needs to be in this order.